spec/requests/meetings/google_calendar_event_spec.rb

```diff
@@ -62,15 +71,17 @@
 
   describe '#destroy' do
     let!(:meeting_member) { create(:meeting_member, user_id: user.id, meeting_id: meeting.id, google_cal_event_id: calendar_event.id) }
-    let!(:remove_response) { double }
```

Comment:
This variable doesn't seem to be used anywhere so I think this is dead code ☠️ ⚰️